Shared/Router/MoviesRouter.swift

```diff
@@ -0,0 +1,4 @@
+public protocol MoviesRouter {
+    func root()
```

Comment:
In a bigger project it is possible to have more than one router and separate by contexts in common.